lib/python/rose/macro.py

```diff
@@ -246,8 +246,8 @@ def _get_config_sections(self, config_data):
             for key, node in config_data.value.items():
                 if isinstance(node.value, dict):
                     sections.append(key)
-                elif "" not in sections:
```

Comment:
There may not be any top-level options in a configuration, so the null section may not exist.